### DISCUSSION TITLE
how can i call ketcher-core with vue or html

### DISCUSSION THREAD

**Discussion by niuzhenjiang** [epam/ketcher#1218] [Q&A] [answered]
I am trying to use vue or html to call the method of ketcher-core, but it does not work, and there is no sample code to learn, can someone tell me the instructions for using vue to embed the ketcher editor? It would be best if you have the code, thanks a lot

**Comment by AndreiMazol** [comment, marked as answer]
Hi @niuzhenjiang ,

this moment ketcher-core is not ready for use in Vue apps. It should be done in scope of the nearest releases. This moment only injection as IFame is available for non-react projects.

Regards,
Andrei
